src/sg/gov/moh/healthcert/1.0/interim-pdt-healthcert-schema.json

```diff
@@ -0,0 +1,437 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "type": "object",
+  "title": "Interim PDT HealthCert Schema",
+  "required": ["id", "name", "validFrom", "fhirVersion", "fhirBundle", "issuers", "$template"],
+  "properties": {
+    "id": {
+      "type": "string",
+      "description": "Internal reference, usually serial number, of this document",
+      "examples": ["TEST001"]
+    },
+    "name": {
+      "type": "string",
+      "examples": ["HealthCert"]
+    },
+    "validFrom": {
+      "type": "string",
+      "format": "date-time",
```

Comment:
Noted that this was "date" in https://github.com/Open-Attestation/schemata/blob/master/src/sg/gov/moh/healthcert/1.0/schema.json. Just need to make sure the issued version has time too.

Reply:
The unwrapped PDT HealthCert uses `date`: https://github.com/Notarise-gov-sg/api-notarise-healthcerts/blob/master/test/fixtures/example_healthcert_with_nric_unwrapped.json#L4

While the Notarised unwrapped PDT HealthCert uses `date-time`: https://github.com/Notarise-gov-sg/api-notarise-healthcerts/blob/master/test/fixtures/example_notarized_healthcert_with_nric_unwrapped.json#L4

Not too sure which one to follow

Reply:
Will stick to `date-time`